openfl/federated/data/loader_fe.py

```diff
@@ -3,13 +3,17 @@
 
 """FastEsitmatorDataLoader module."""
 
+from typing import Tuple
+
+from fastestimator import Pipeline
```

Comment:
To avoid import errors you may try adding `fastestimator` condition in https://github.com/intel/openfl/blob/ec17e6f5bde04ba3e9ae4939c5e8aca1eb714e02/openfl/federated/data/__init__.py#L26